breaking-changes-log.md

```diff
@@ -8,6 +8,635 @@ parent:
+
+```rust
+/* BEFORE */
+let gas_price = tx_gas_price();
+
+/* AFTER */
+let tip = tx_tip();
+```
+
+Release: [Sway v0.54.0](https://github.com/FuelLabs/sway/releases/tag/v0.54.0)
+
+The `forc-client` and `forc-tx` plugins now take `u16` instead of `u8` for witness index types.
+
+### TS-SDK
+
+Release [v0.83.0](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.83.0)
+
+`BaseAssetId` is no longer exported by `fuels`. It can be fetched from a `Provider`.
+
+```ts
+/* BEFORE */
+import { BasedAssetId } from "fuels";
+
+/* AFTER */
+const provider = await Provider.create(FUEL_NETWORK_URL);
+const baseAssetId = provider.getBaseAssetId();
+```
+
+`TransactionRequest.addCoinOutput` and `TransactionRequest.addChangeOutput` now requires an `assetId`, it no longer defaults to the `BaseAssetId`.
+
+`TransactionRequest.fundWithFakeUtxos` now requires passing the `baseAssetId` as a function parameter. This is the only function that is base asset aware, so that it can be used specifically to estimate the transaction cost.
+
+`CoinQuantityLike` now requires an `AssetId`. Previously most of it's usages would default to the `BaseAssetId`, as this must now be fetched, so it must be passed to the type.
+
+```ts
+/* BEFORE */
+let coin: CoinQuantityLike = [1000];
+coin = { amount: 1000 };
+
+/* AFTER */
+const assetId = "0x..";
+let coin: CoinQuantityLike = [1000, assetId];
+coin = { amount: 1000, assetId };
+```
+
+`gasPrice` is calculated by the VM so we do not need use it anymore.
+
+```ts
+/* BEFORE */
+await factory.deployContract({ gasPrice });
+
+/* AFTER */
+await factory.deployContract();
+```
+
+`PolicyType.GasPrice` is now `PolicyType.Tip`.
+
+The `Account.fund` function parameters changed.
+
+```ts
+/* BEFORE */
+async fund<T extends TransactionRequest>(
+    request: T,
+    coinQuantities: CoinQuantity[],
+    fee: BN,
+    inputsWithEstimatedPredicates: TransactionRequestInput[],
+    addedSignatures?: number
+): Promise<T>
+
+/* AFTER */
+export type EstimatedTxParams = {
+  minFee: BN;
+  estimatedPredicates: TransactionRequestInput[];
+  addedSignatures: number;
+  requiredQuantities: CoinQuantity[];
+}
+async fund<T extends TransactionRequest>(request: T, params: EstimatedTxParams): Promise<T>
+```
+
+GraphQL URL now includes a versioning path: `http://127.0.0.1:4000/v1/graphql`.
+
+`calculateTransactionFee` now requires `tip`, `maxGasPerTx`, and `gasPrice`. Also `gasUsed` is not used anymore.
+
+```ts
+/* BEFORE */
+const { fee } = calculateTransactionFee({
+  gasUsed,
+  rawPayload,
+  consensusParameters: {
+    gasCosts,
+    feeParams: {
+      gasPerByte,
+      gasPriceFactor,
+    },
+  },
+});
+
+/* AFTER */
+const { fee } = calculateTransactionFee({
+  gasPrice, // new
+  tip, // new
+  consensusParameters: {
+    maxGasPerTx, // new
+    gasCosts,
+    feeParams: {
+      gasPerByte,
+      gasPriceFactor,
+    },
+  },
+  rawPayload,
+});
+```
+
+`AssetId` and `EvmAddress` property `value` was renamed to `bits`.
+
```

Comment:
Hmm this was due to the `forc` upgrade rather than a design decision our side, should it reference the sway breaking change?